app/user/rest.py

```diff
@@ -454,6 +457,26 @@ def send_support_email(user_id):
     return jsonify({"status_code": status_code}), 204
 
 
+@user_blueprint.route('/<uuid:user_id>/contact-request', methods=['POST'])
+def send_contact_request(user_id):
```

Comment:
This HTTP method is not tested and it's important to have tests for this controller as it does non-trivial things and is in charge of adding the required tags to skip urgent escalation paths on Freshdesk